nncf/quantization/passes.py

```diff
@@ -23,35 +24,38 @@
 def transform_to_inference_graph(
     nncf_graph: NNCFGraph,
     shapeof_metatypes: List[OperatorMetatype],
```

Comment:
metatypes can be `Optional`, to avoid call with empty args.

Reply:
Yeah, but it is always a list entity. In case there are no target metatypes metatypes list is empty, so there is no signature violation here

Reply:
Outdated